packages/eas-cli/src/commands/update/configure.ts

```diff
@@ -39,297 +25,27 @@ export default class UpdateConfigure extends EasCommand {
-
-    if (!isExpoUpdatesInstalledOrAvailable(projectDir, exp.sdkVersion)) {
-      await installExpoUpdatesAsync(projectDir);
-    }
 
-    const workflows = await resolveWorkflowPerPlatformAsync(projectDir);
+    Log.log(
+      '💡 The following process will configure your project to run EAS Update. These changes only apply to your local project files and you can safely revert them at any time.'
+    );
 
-    const updatedExp = await configureAppJSONForEASUpdateAsync({
-      projectDir,
-      projectId,
+    await ensureEASUpdatesIsConfiguredAsync(graphqlClient, {
       exp,
-      platform,
-      workflows,
-    });
-
-    await configureNativeFilesForEASUpdateAsync({
-      projectDir,
       projectId,
-      exp: updatedExp,
+      projectDir,
       platform,
-      workflows,
-      graphqlClient,
     });
 
-    Log.addNewLineIfNone();
-    Log.warn(
-      `All builds of your app going forward will be eligible to receive updates published with EAS Update.`
-    );
-
     Log.addNewLineIfNone();
     Log.log(`🎉 Your app is configured to run EAS Update!`);
   }
 }
-
-type ConfigureForEASUpdateArgs = {
-  projectDir: string;
-  exp: ExpoConfig;
-  platform: RequestedPlatform;
-  workflows: Record<Platform, Workflow>;
-  projectId: string;
-};
-
-export async function configureAppJSONForEASUpdateAsync({
-  projectDir,
-  exp,
-  platform,
-  workflows,
-  projectId,
-}: ConfigureForEASUpdateArgs): Promise<ExpoConfig> {
-  // this command is non-interactive in the way it was designed
-  const easUpdateURL = getEASUpdateURL(projectId);
-  const updates = { ...exp.updates, url: easUpdateURL };
-
-  const androidDefaultRuntimeVersion =
-    workflows.android === Workflow.GENERIC
-      ? DEFAULT_BARE_RUNTIME_VERSION
-      : DEFAULT_MANAGED_RUNTIME_VERSION;
-  const iosDefaultRuntimeVersion =
-    workflows.ios === Workflow.GENERIC
-      ? DEFAULT_BARE_RUNTIME_VERSION
-      : DEFAULT_MANAGED_RUNTIME_VERSION;
-
-  const newAndroidRuntimeVersion =
-    exp.android?.runtimeVersion ?? exp.runtimeVersion ?? androidDefaultRuntimeVersion;
-  const newIosRuntimeVersion =
-    exp.ios?.runtimeVersion ?? exp.runtimeVersion ?? iosDefaultRuntimeVersion;
-
-  let newConfig: Partial<ExpoConfig>;
-  let newConfigOnlyAddedValues: Partial<ExpoConfig>;
-  switch (platform) {
-    case RequestedPlatform.All: {
-      if (isRuntimeEqual(newAndroidRuntimeVersion, newIosRuntimeVersion)) {
-        newConfig = {
-          runtimeVersion: newAndroidRuntimeVersion,
-          android: {
-            ...exp.android,
-            runtimeVersion: undefined,
-          },
-          ios: { ...exp.ios, runtimeVersion: undefined },
-          updates,
-        };
-        newConfigOnlyAddedValues = {
-          runtimeVersion: newAndroidRuntimeVersion,
-          ...(exp.android && 'runtimeVersion' in exp.android
-            ? {
-                android: {
-                  runtimeVersion: '<remove this key>',
```

Comment:
looks like the new version of this doesn't have this in the manual instruction?

Reply:
I think you are right. I can compensate for that in the `warnEASUpdatesManualConfig`. We can iterate over the `modifyConfig` and replace `undefined` with this string.

Reply:
Done